### PR TITLE
Added Windows 95 style buttons in the Specials

### PR DESCRIPTION
**Pull request by anonyda**
Hey, I created another button type, which closes Issue #433 
These are Windows 95 style vintage buttons.
I am using a font from Google Fonts.
Do let me know how you think of it!

### CODE REVIEW

File: src/components/special/_win95.less [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  box-shadow: 0 0 0 black;
+  transition: all 0.2s;
+}
+
+.win95-btn:last-child {
+  margin: 0;
+}
+
+.win95-btn:hover {
+  box-shadow: 0.4rem 0.4rem 0 black;
```
Comment (shahednasser) [inline src/components/special/_win95.less:23]: Reduce translate and box-shadow on hover I think it is a little too much

**Review by shahednasser — CHANGES_REQUESTED** [review]
Is using this google font necessary? It doesn't really make it look more like a windows 95 button. Either choose a different font or don't add one.

Also in the less file can you please use less syntax? i.e, instead of repeating `.win95-btn` over and over again, place everything related to it inside it once and refer to the button with "&". Please refer to Less on more info on this.

**Comment by shahednasser** [comment]
I suggest using this font instead: https://fonts.google.com/specimen/VT323?category=Monospace

File: src/components/special/_win95.less (hunk 1 above)
Comment (anonyda) [reply src/components/special/_win95.less]: thanks for the feedback! I made the changes.

**Comment by shahednasser** [comment]
Looks good thank you!

**Pull request merged**